platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1134,4 +1138,30 @@ String getFontFallbackFilePath(int importance, int weightHint) {
         return fontFileParser.getFontFallback(importance, weightHint);
     }
 
+    // Tmp Asset Path
+    // ==============
+    String copyAssetToTmpPath(String path) throws IOException {
```

Comment:
We shouldn't throw the exception here this is called from JNI code otherwise we would crash if the application fails to find this mbtile file.